tests/system/robot/startupShutdownNVDA.py

```diff
@@ -77,8 +82,12 @@ def quits_from_menu(showExitDialog=True):
 		_builtIn.sleep(1)  # the dialog is not always receiving the enter keypress, wait a little for it
 		spy.emulateKeyPress("enter", blockUntilProcessed=False)  # don't block so NVDA can exit
 
-	_process.wait_for_process(_nvdaProcessAlias, timeout="3 sec")
-	_process.process_should_be_stopped(_nvdaProcessAlias)
+	_blockUntilConditionMet(
+		getValue=lambda: _getNvdaMessageWindowhandle() == 0,

```

Comment:
I think the following is clearer:

```suggestion
		getValue=lambda: not _nvdaIsRunning(),
```